src/ControlSystem/SettleToConstant.cpp

```diff
@@ -0,0 +1,65 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "ControlSystem/SettleToConstant.hpp"
+
+#include <algorithm>
+#include <utility>
+
+#include "ErrorHandling/Error.hpp"
+#include "Utilities/MakeArray.hpp"
+
+FunctionsOfTime::SettleToConstant::SettleToConstant(
+    std::array<DataVector, 3> initial_func_and_derivs, double t0,
+    double decay_time) noexcept : match_time_(t0),
+                                  decay_freq_(1.0 / decay_time) {
+  // C = -dtF-dt2F*decay_time;
+  // B = (C-dtF)*decay_time;
+  // A = F-B;
+  coef_c_ =
+      -initial_func_and_derivs[1] - initial_func_and_derivs[2] * decay_time;
+  coef_b_ = (coef_c_ - initial_func_and_derivs[1]) * decay_time;
+  coef_a_ = initial_func_and_derivs[0] - coef_b_;
+}
+
+template <size_t MaxDerivReturned>
+std::array<DataVector, MaxDerivReturned + 1>
+FunctionsOfTime::SettleToConstant::func_and_derivs(const double t) const
+    noexcept {
+  if (not std::isfinite(t)) {
```

Comment:
static_assert(MaxDerivReturned < 3, "some message");

Reply:
I removed the finite(t) check (which is what was done in SpEC). 
But, is the static_assert necessary to add, given that  func_and_derivs is a private member function?

Reply:
Yes, nothing wrong with enforcing correctness in private members :)